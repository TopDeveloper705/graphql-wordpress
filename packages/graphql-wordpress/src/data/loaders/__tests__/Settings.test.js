import getSettingsLoaders from 'data/loaders/Settings';

const Settings = getSettingsLoaders();

describe('Test Settings loader', () => {
  test('Load settings', async () => {
    const settings = await Settings.load();
    expect(settings.getID()).toMatchSnapshot();
    expect(settings).toMatchSnapshot();
  });
});
