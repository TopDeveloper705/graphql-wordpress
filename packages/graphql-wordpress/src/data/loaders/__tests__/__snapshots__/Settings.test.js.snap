// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Settings loader Load settings 1`] = `"U2V0dGluZ3M6bWU="`;

exports[`Test Settings loader Load settings 2`] = `
Settings {
  "slug": "graphql/v1/settings",
}
`;
